src/Components/Body/Chat/NewMessage.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  AiOutlineVideoCameraAdd,
+  AiOutlineSend,
+  AiOutlineExclamation
+} from "react-icons/ai";
+import { MdTextFormat, MdCelebration } from "react-icons/md";
+import { SiSololearn } from "react-icons/si";
+import { ImAttachment } from "react-icons/im";
+import { MdOutlineLoop, MdOutlineGifBox, MdApproval } from "react-icons/md";
+import { IoIosMore } from "react-icons/io";
+import { BsCalendar2Plus, BsCameraReels } from "react-icons/bs";
+import { GrEmoji } from "react-icons/gr";
+import { BiRightTopArrowCircle } from "react-icons/bi";
+import { GiNotebook } from "react-icons/gi";
+import { useState } from "react";
+
+type NewMessageProps = {
+  handleSendMessage: (message: string) => void
+}
+export default function NewMessage({ handleSendMessage }: NewMessageProps) {
```

Comment:
Props should never have callback name prefixed with `handle` we should use `on` instead.

handleSendMessage <> onSendMessage

The actual handlers which handles this event should be prefixed with handle.
Example:

While defining handler function.
`const handleClick = () => { // do something };`

While passing this callback to child components:
`onClick={handleClick}`